contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Issue Triage Guidelines
+
+## Purpose
+
+Speed up issue management.
+
+The Kubernetes issues are listed at https://github.com/kubernetes/kubernetes/issues and are identified with labels. For example, an issue that belongs to SIG Network will eventually be set to label `sig/network`. New issues will start out without any labels. The detailed list of labels can be found at https://github.com/kubernetes/kubernetes/labels. While working on triaging issues you may not have privilege to assign specific label (e.g. `triaged`) and in that case simply add a comment in the issue with your findings.
+
+## Scope
+
+These guidelines serves as a primary document for triaging incoming issues to Kubernetes. SIGs and projects are encouraged to either use these guidelines, or use this as a starting point if necessary. For example if your SIG has specific triaging needs, extend these guidelines.
+
+## Using the bot
+
+Most people do not have the ability to interact with the github issues and labels, for that we use a bot to manage labelling and triaging. The bot has a set of [commands and permissions](https://github.com/kubernetes/test-infra/blob/master/commands.md) and this document will cover the basic ones.
+
+
+## Determine if it’s a support request
+
+Sometimes users ask for support requests in issues; these are usually support requests for people who need help configuring some aspect of Kubernetes. These should be directed to our [support structures](https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-user-support.md) and then closed. Only authors and assignees can `/close` an issue. Also, if the issue is clearly abandoned or in the wrong place you can `/assign` to have it assigned to yourself, and then `/close`.
+
+## Validate if the issue is real
+
+Validate if the problem is real by reproducing it. If reproducible, move to the next step of finding a right SIG. You may need to contact the issue reporter in the following cases:
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate.
+* If you can not reproduce the issue, contact the issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If the issue is non-trivial to reproduce, work with issue reporter or move to next step of finding a right SIG, and let SIG know of your findings.
+* If you do not get a response in 15 days, then close the issue with appropriate comment.
```

Comment:
@xiangpengzhao true but it can always be reopened so I prefer it :). We had some discussion and we came up with little aggressive dead line. I would bring it up in Contrib SIG meeting or we can discuss it here if you want. /cc @castrojo